package.json

```
@@ -31,6 +31,13 @@
   "main": "./dist/lib/server-impl.js",
   "scripts": {
     "start": "TZ=UTC node ./dist/server.js",
+    "build:backend": "copyfiles -u 1 src/mailtemplates/**/*.mustache dist/ && tsc --pretty --strictNullChecks false",
+    "build:frontend": "cd ./frontend && yarn run build",
+    "build": "yarn run build:frontend && yarn run build:backend",
+    "dev:backend": "TZ=UTC NODE_ENV=development tsc-watch --strictNullChecks false --onSuccess \"node dist/server-dev.js\"",
+    "dev:frontend": "wait-on tcp:4242 && cd ./frontend && yarn run dev",
```

Nice solution with `wait-on` and concurrently!! 
```suggestion
    "dev:frontend": "wait-on tcp:4242 && yarn --cwd ./frontend run dev",
```